tests/modules/test_upload_contacts_from_file.py

```diff
@@ -235,3 +244,13 @@ def test_parse_contact_time_references_real_datetimes(self):
 			parse_contact_time_references(time1))
 		self.assertEqual(datetime(2017, 6, 16, 18, 51, 28, tzinfo=timezone.get_default_timezone()),
 			parse_contact_time_references(time2))
+
+	def test_parse_contact_time_references_fake_datetimes(self):
+		fake_time1 = "6/12/2017 25:00:03 PM"
+		fake_time2 = "6/30/2017 6:51:28 PM"
+
+		with self.assertRaises(ValueError):
+			parse_contact_time_references(fake_time1)
+			parse_contact_time_references(fake_time2)
```

Comment:
I think this test only tests if one of the two produces a `ValueError`, not both. I think you need separate `assertRaises` steps.